main.c

```diff
@@ -93,6 +93,10 @@ int main(void) {
 }
 
 void setup() {
+	uint8_t retcode;         // variable to store retcode
+	uint8_t msg_str[40];     // variable to store a message string
+	uint8_t hmc_retcode; 	 // variable to store retcode hmc module
+	uint8_t hmc_msg_str[40]; // variable to store a message string for retcode hmc module
```

Comment:
Do not use TABs! It's prohibited.
You can customize your IDE to replace tabs with spaces. We use 1 tab = 4 spaces mapping.